src/app/client/partials/commitStreamAdmin.html

```diff
@@ -484,36 +484,23 @@
     <h1>{{getHeading()}}</h1>
     <div class='well well-sm' ng-if='editAllowed()'>
       <div class='vcs-families'>
-        <div class='vcs-family'>
-          <button type='button' ng-click='setSelectedButton("GitHub")' ng-mouseenter='familyHover("GitHub")' ng-mouseleave='familyHover("")' class='{{getClass("GitHub")}}'>GitHub&nbsp;&nbsp;<img ng-src='{{serviceUrl}}/{{familyIsSelectedIcon("GitHub")}}' /></button>&nbsp;
-        </div>
-        <div class='vcs-family'>
-          <button type='button' ng-click='setSelectedButton("GitLab")' ng-mouseenter='familyHover("GitLab")' ng-mouseleave='familyHover("")' class='{{getClass("GitLab")}}'>GitLab&nbsp;&nbsp;<img ng-src='{{serviceUrl}}/{{familyIsSelectedIcon("GitLab")}}' style="height:32px;" /></button>
-        </div>
-        <div class='vcs-family'>
-          <button type='button' ng-click='setSelectedButton("Bitbucket")' ng-mouseenter='familyHover("Bitbucket")' ng-mouseleave='familyHover("")' class='{{getClass("Bitbucket")}}'>Bitbucket<img ng-src='{{serviceUrl}}/{{familyIsSelectedIcon("Bitbucket")}}' style="height:32px;" /></button>
-        </div>
-        <div class='vcs-family'>
-          <button type='button' ng-click='setSelectedButton("VSTS")' ng-mouseenter='familyHover("VSTS")' ng-mouseleave='familyHover("")' class='{{getClass("VSTS")}}'>VSTS&nbsp;&nbsp;&nbsp;&nbsp;<img ng-src='{{serviceUrl}}/{{vstsFamilyIsSelectedIcon("VSTS")}}' style="height:32px;" /></button>
-          <div class='vcs-vso-options' ng-init="value='VsoGit'" ng-if='showVSTSChoices()'>
-              <label><input type="radio" name="vsoFamily" ng-model="value" value="VsoGit" ng-change='vstsSelection(value)'/><img ng-src='{{vsoIcon("vso-git")}}' style="height:24px; margin-left:6px;" />&nbspGit</label>
-              <br/>
-              <label><input type='radio' name="vsoFamily" ng-model="value" value="VsoTfvc" ng-change='vstsSelection(value)'><img ng-src='{{vsoIcon("vso-tfvc")}}' style="height:24px; margin-left:6px;" />&nbspTFVC</label>
+        <div class='vcs-family' ng-repeat="familyInfo in getFamiliesArr() | orderBy:'displayOrder'">
```

Comment:
I've been wanting to `ng-repeat` this section for years now :D Thanks again for making this part data-bound instead of hard-coded!